docs/index.asciidoc

```diff
@@ -143,7 +143,7 @@ This plugin supports the following configuration options plus the <<plugins-{typ
 | <<plugins-{type}s-{plugin}-query_template>> |<<string,string>>|No
 | <<plugins-{type}s-{plugin}-result_size>> |<<number,number>>|No
 | <<plugins-{type}s-{plugin}-retry_on_failure>> |<<number,number>>|No
-| <<plugins-{type}s-{plugin}-retry_on_status>> |<<number,number list>>|No
+| <<plugins-{type}s-{plugin}-retry_on_status>> |<<number,number>>|No
```

Comment:
The intent here was to have the text of the link say "number list" because it takes a list of numbers.

Reply:
What about `result_on_status_>>` --> `retry_on_status>>`? This change aligns with  labeling in descriptions section.

Reply:
`<<number,number list>>` will work for plugins generated for the Logstash Plugin Reference, but will fail in the Versioned Plugin Reference (VPR). Why? Because the abbreviated anchor format (`<<internal-anchor>>`) works only _within the same book_.  Outside of the context of a particular book, that format has no meaning.  

Restated:
* **LSR.** This format works in the LSR because the [topic](https://www.elastic.co/guide/en/logstash/current/configuration-file-structure.html#number) is present in the Logstash Reference. 
* **VPR.** The reason the format for our typical [Value types](https://www.elastic.co/guide/en/logstash/current/configuration-file-structure.html#plugin-value-types) works in the VPR at all is because of gsubs in the [VPR docgen script](https://github.com/elastic/docs-tools/blob/main/versioned_plugins.rb):

   <img width="937" alt="Screen Shot 2023-03-10 at 2 40 32 PM" src="https://user-images.githubusercontent.com/35154725/224412771-1dac9dc8-9421-4f7e-badb-a0c4c0737fc6.png">
   
So when you modify the expected format to be `<<number,number list>>`, it doesn't match the gsub `<<number,number>>` so it renders literally, and fails docs CI in the [VPR docgen PR.](https://github.com/elastic/logstash-docs/pull/1478)

Reply:
Related: 
If a `number list` is something we expect to use more frequently in the future, we should update docs for to add `number list` as a value type.  OR expand the current entry for number to show an example/description of a number list.

Reply:
:thinking: -> https://github.com/elastic/docs-tools/pull/86

Reply:
Not sure we want to make it so easy to deviate from the standard.  Instead we should vet and update links and gsubs to be more appropriate to our input types. 

> If a number list is something we expect to use more frequently in the future, we should update docs for to add number list as a value type. OR expand the current entry for number to show an example/description of a number list. The current description and example is for a single entry.